redaxo/src/addons/phpmailer/composer.json

```diff
@@ -1,3 +1,3 @@
 {
-    "require": { "phpmailer/phpmailer": "~5.2" }
+    "require": { "phpmailer/phpmailer/src": "~5.2" }
```

Comment:
Das /src hier muss sicher weg. Und die version hinten auf ^6.0 ändern

Reply:
manuell. ich muss mich erst noch mit composer beschäftigen. Aber ich sehe den Fehler jetzt

Reply:
Loesche das vendor verzeichnis, ändere die composer.json und installieren composer auf deinem Computer. Dann mit `composer install` in der commandline die lib installieren (dazu musst du in dem root verzeichnis des addons stehen; also dort wo die composer.json liegt)

Reply:
Gibt's n Weg composer problemlos auf dem Mac zu installieren. Habe es mit Homebrew versucht. - Keine chance